### PR TITLE
Adding Anonymous Sign Ins

### PR DESCRIPTION
**Pull request by dishwasher-detergent**

### CODE REVIEW

File: src/account/useAnonSignIn.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { Models } from "appwrite";
+import { useAppwrite } from "react-appwrite";
+
+function useAnonSignIn() {
```
Comment (sanny-io) [inline src/account/useAnonSignIn.ts:7]: Can you rename it to `useAnonymousSignIn`? I think it is better to name things with their full name and to only shorten when it makes the most sense, like with OAuth for example.

File: src/account/index.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-export * from './useOAuth2SignIn'
-export * from './useSignOut'
\ No newline at end of file
+export * from "./types"
+export * from "./useAccount"
+export * from "./useEmailSignIn"
+export * from "./useEmailSignUp"
+export * from "./useAnonSignIn"
+export * from "./useOAuth2SignIn"
+export * from "./useSignOut"
```
Comment (sanny-io) [inline src/account/index.ts:7]: Can you change these to use single quotes `'` to match the rest of the project and remove semicolons throughout? I see you made a commit to match, but it looks like you may have missed these.

**Review by sanny-io — CHANGES_REQUESTED** [review]
Thank you for your PR 🙂

Just a few changes would be good: renaming the function to not be abbreviated, using single quotes `'`, and removing semicolons.

We will probably need some sort of style plugin to keep everything as consistent as possible. I will create an issue for that.

Thanks for your contribution bro 👍

File: src/account/index.ts (hunk 2 above)
Comment (dishwasher-detergent) [reply src/account/index.ts:7]: Will do! I used the default prettier configs so I disabled prettier and tried to go back and revert the changes it made. Guess I missed a few :P

**Review by sanny-io — APPROVED** [review]
Looks good! Thanks again 🙂

**Pull request merged**